ecosystem/gstreamer_plugin/README.md

```diff
@@ -0,0 +1,408 @@
+| rx-fps              | uint    | Framerate of the video.                             | [gst_mtl_supported_fps](#231-supported-video-fps-codes-gst_mtl_supported_fps) | 0 |
+| rx-framebuff-num    | uint    | Number of framebuffers to be used for transmission. | 0 to 8                     | 3             |
+| rx-width            | uint    | Width of the video.                                 | 0 to G_MAXUINT             | 1920          |
+| rx-height           | uint    | Height of the video.                                | 0 to G_MAXUINT             | 1080          |
+| rx-interlaced       | boolean | Whether the video is interlaced.                    | TRUE/FALSE                 | FALSE         |
+| rx-pixel-format     | string  | Pixel format of the video.                          | `v210`, `YUV444PLANAR10LE` | `v210`        |
+
+#### 3.2.2. Preparing output path
+
+In our pipelines, we will use the `$OUTPUT` variable to hold the path to the video.
+
+```bash
+export OUTPUT="path_to_the_file_we_want_to_save"
+```
+
+#### 3.2.3. Pipline example for multicast with `y210` format
+
+To run the `mtl_st20p_rx` plugin, use the following command to specify the input parameters of the incoming stream.
+
+> **Warning**: To receive data, ensure that a transmission is running with the same parameters on the `239.168.75.30` address.
+
+```bash
+# If you don't know how to find the VFIO PCI address of your device
+# refer to Media-Transport-Library/doc/run.md
+export VFIO_PORT_R="pci_address_of_the_device"
+
+# Run the receiver pipeline y210 FHD 60fps on port 20000
+gst-launch-1.0 mtl_st20p_rx rx-queues=4 udp-port=20000 payload-type=112 dev-ip="192.168.96.3" ip="239.168.75.30" dev-port=$VFIO_PORT_R rx-pixel-format=v210 rx-height=1080 rx-width=1920 rx-fps=25 ! \
+filesink location=$OUTPUT --gst-plugin-path $GSTREAMER_PLUGINS_PATH
+```
+
+This command sets up the receiver pipeline with the specified parameters and saves the received video to the specified output path.
+
+## 4. SMPTE ST 2110-30 Raw audio plugins
+
+Audio plugins for MTL that are able to send, receive synchronous raw audio via the MTL pipeline API.
+
+### 4.1. Running the SMPTE ST 2110-30 transmission plugin mtl_st30p_tx
+
+#### 4.1.1. Supported Parameters and Pad Capabilities
+
+The `mtl_st30p_tx` plugin supports the following pad capabilities:
+
+- **Formats**: `S16LE`, `S24LE`, `S32LE`
+- **Sample Rate Range**: 44100, 48000, 96000
+- **Channels Range**: 1 to 8
+
+**Arguments**
+| Property Name       | Type   | Description                                           | Range                   | Default Value |
+|---------------------|--------|-------------------------------------------------------|-------------------------|---------------|
+| retry               | uint   | Number of times the MTL will try to get a frame.      | 0 to G_MAXUINT          | 10            |
```

Comment:
I think we have a bug, `retry` parameter is not "installed" with `g_object_class_install_property`. Same with  tx-samplerate and tx-channels, they are only negotiated with previous plugin in pipeline, can't be directly set and I guess they shouldn't.

Reply:
Oh right i only added it to the ST 20 
good catch 

its fine as it is i can also add with documentation :?